src/vstart.sh

```diff
@@ -350,40 +352,38 @@ parse_secondary_devs() {
 
 # Auxiliar function to prepare the CPU cores to pin Seastar reactors
 prep_balance_cpu() {
-    local crimson_smp=$1
-    local balance_strategy=$2
-    local in_file_name="/tmp/numa_args_${balance_strategy}.out"
-    local out_file_name="/tmp/numa_nodes.json"
-    local log_file_name="/tmp/numa_bal_${balance_strategy}.log"
-    local cmd
-
-    # Check the table is empty
-    if [ "${#cpu_table[@]}" -eq 0 ]; then
-        # Ensure the file with the CPU mappings exist
-        if [ ! -f ${in_file_name} ]; then
-            debug echo "lscpu --json > ${out_file_name}"
-            lscpu --json > ${out_file_name}
-            MY_CPUS=$(taskset -acp $$ | awk -F : '{print $2}')
-            cmd="python3 ${CEPH_DIR}/../src/tools/contrib/balance_cpu.py -o ${CEPH_NUM_OSD}\
-                -r ${crimson_smp}  -b ${balance_strategy} -u ${out_file_name} -t ${MY_CPUS} > ${in_file_name}"
-            debug echo "$cmd"
-            eval "$cmd" >> ${log_file_name}
-        fi
+    if [ -z $crimson_balance_cpu ] || [ "${crimson_balance_cpu}" == "none" ] ; then
```

Comment:
nit: I think we should fail here on any entry which is not `osd` or `socket` (instead of only on "none").

"none" and "0" and mistakes like "osdsockettt" will all lead to not assigning cpus.

Otherwise we would try to assign cpus with faulty `-b` args.